### PR TITLE
Added documentation for working with insights upload command (#9)

### LINKED ISSUE #9: Document the new qpc command for insights upload
## Feature description:

Document the new commands for insights upload & insights upload-status.

### PR DESCRIPTION
**Pull request by myersCody**
Closes #9

### CODE REVIEW

File: docs/source/working_with_insights.rst [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Working with Insights
+---------------------
+If you would like to use Quipucords to upload data to Red Hat Insights, some additional external setup is required. The qpc commands related to Insights expect the Insights client to be installed. More information about setting up the Insights client can be found here: https://access.redhat.com/products/red-hat-insights#getstarted.
+
+
+Uploading To Insights
+^^^^^^^^^^^^^^^^^^^^^
+When a scan completes, it returns a report identifier that can then be used to upload a deployments report to Insights. The deployments report process runs steps to merge the facts found during the inspection of various hosts that are contacted during the scan. To generate and upload a deployments report to insights, run the ``qpc insights upload`` command and specify the report identifier.
```
Comment (abaiken) [inline docs/source/working_with_insights.rst]: If we decide to add the option to upload a report based on scan job id, I would reword this to say:

Use the ``qpc insights upload`` command to upload a deployments report to Red Hat Insights. You can upload a report to Insights using the associated report identifier or scan job identifier for the scan that is used to generate the report. 

```qpc insights upload (--scan-job scan_job_identifier | --report report_identifier )```

File: docs/source/working_with_insights.rst [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+For example, to create and upload the deployments report for a scan with report identifier ``1``, you would enter the following command::
```
Comment (abaiken) [inline docs/source/working_with_insights.rst]: Change this to say: 

For example, to create and upload the deployments report with a report identifier of ``1``, you would enter the following command::

**Comment by abaiken** [comment]
We should connect this rst file to readthedocs via the index.rst file

File: docs/source/working_with_insights.rst [hunk 3]
```diff
@@ -0,0 +1,12 @@
+Working with Insights
+---------------------
+If you would like to use Quipucords to upload data to Red Hat Insights, some additional external setup is required. The qpc commands related to Insights expect the Insights client to be installed. More information about setting up the Insights client can be found here: https://access.redhat.com/products/red-hat-insights#getstarted.
```
[the remark this answers is not in the document]
Comment (kholdaway) [reply docs/source/working_with_insights.rst]: 👍

File: docs/source/working_with_insights.rst (hunk 1 above)
Comment (kholdaway) [reply docs/source/working_with_insights.rst]: We should to be consistent.

File: docs/source/working_with_insights.rst (hunk 2 above)
Comment (kholdaway) [reply docs/source/working_with_insights.rst]: 👍

File: docs/source/working_with_insights.rst [hunk 4]
```diff
@@ -0,0 +1,14 @@
+Working with Insights
+---------------------
+In order to use Quipucords to upload data to Red Hat Insights, the Insights client must be installed. To install and setup the Insights client, visit https://access.redhat.com/products/red-hat-insights#getstarted and follow the instructions.
+
+
+Uploading To Insights
+^^^^^^^^^^^^^^^^^^^^^
+Use the qpc insights upload command to upload a deployments report to Red Hat Insights. You can upload a report to Insights using the associated report identifier or scan job identifier for the scan that is used to generate the report::
```
Comment (abaiken) [inline docs/source/working_with_insights.rst]: put "qpc insights upload" in back tics so it becomes ``qpc insights uplod``

File: docs/source/working_with_insights.rst [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+
+  # qpc insights upload (--scan-job scan_job_identifier | --report report_identifier )
```
Comment (abaiken) [inline docs/source/working_with_insights.rst]: remove the # here

File: docs/source/working_with_insights.rst [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+
+For example, to create and upload the deployments report with a report identifier of ``1``, you would enter the following command::
+
+  # qpc insights upload --report 1
```
Comment (abaiken) [inline docs/source/working_with_insights.rst]: remove the #

**Comment by myersCody** [comment]
This issue was moved to quipucords: https://github.com/quipucords/quipucords/issues/1689
